xray/commands/audit/sca/npm/npm.go

```diff
@@ -28,10 +29,18 @@ func BuildDependencyTree(npmArgs []string) (dependencyTrees []*xrayUtils.GraphNo
 	if err != nil {
 		return
 	}
-	npmArgs = addIgnoreScriptsFlag(npmArgs)
-
+	treeDepsParam := biutils.NpmTreeDepListParam{
+		Args: addIgnoreScriptsFlag([]string{}),
+	}
+	if params != nil {
+		treeDepsParam = biutils.NpmTreeDepListParam{
+			Args:                 addIgnoreScriptsFlag(params.Args()),
+			IgnoreNodeModules:    params.NpmIgnoreNodeModules(),
+			OverWritePackageLock: params.NpmOverwritePackageLock(),
+		}
+	}
```

Comment:
Let's export this logic to a new private method "createTreeDepsParam".
We must keep the methods in the code as short as possible.